core/templates/pages/exploration-player-page/modals/display-hint-modal.component.ts

```diff
@@ -33,12 +34,12 @@ import { PlayerTranscriptService } from '../services/player-transcript.service';
   templateUrl: './display-hint-modal.component.html'
 })
 export class DisplayHintModalComponent {
-  index: number;
-  COMPONENT_NAME_HINT: string;
-  hint: SubtitledHtml;
-  displayedCard: StateCard;
-  recordedVoiceovers: RecordedVoiceovers;
-  hintContentId: string;
+  index: number | undefined;
+  COMPONENT_NAME_HINT: string = AppConstants.COMPONENT_NAME_HINT;
```

Comment:
I think you can remove this and use `AppConstants.COMPONENT_NAME_HINT` directly below.

Reply:
Done!